index.html

```diff
@@ -232,5 +229,16 @@ <h3 class="lets-connect">LET'S CONNECT</h3>
                 </div>
             </div>
         </section>
+    <section class="contact">
+            <h2>Contact Me</h2>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form action="https://formspree.io/f/xgeqpkgk" method="post">
+            <input type="text" name="text" id="text" placeholder="Name" maxlength="30" required>
+            <input type="email" name="email" id="email" placeholder="Email" required>
+            <textarea name="formMessage" id="message" placeholder="Message" maxlength="500" required></textarea>
+            <button class="form-btn">Get in Touch</button>
```

Comment:
- [ ] as part of the requirements & to match Figma design kindly add the button interactions such as hover, oressed, etc .. like shown in Figma 

![image](https://user-images.githubusercontent.com/89282221/199440298-45ec65d7-b463-4081-96cd-5a85c9a5836c.png)